ext/scheduler/airflow2/resources/base_dag.py

```diff
@@ -182,15 +182,14 @@
 {{- $dependencySchema := $dependency.Job.Task.Unit.Info }}
 
 {{- if eq $dependency.Type $.JobSpecDependencyTypeIntra }}
-wait_{{$dependency.Job.Name | replace "-" "__dash__" | replace "." "__dot__"}} = SuperExternalTaskSensor(
-    external_dag_id="{{$dependency.Job.Name}}",
-    window_size={{$baseWindow.Size.String | quote}},
-    window_offset={{$baseWindow.Offset.String | quote}},
-    window_truncate_to={{$baseWindow.TruncateTo | quote}},
+wait_{{$dependency.Job.Name | replace "-" "__dash__" | replace "." "__dot__"}} = CrossTenantDependencySensor(
```

Comment:
we need to rename CrossTenantDependencySensor to SuperExternalTaskSensor and use that and remove CrossTenantDependencySensor.

Reply:
this change has been done and merged branch with  https://github.com/odpf/optimus/pull/258.

also, deleting this pr for the same reason